architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -147,6 +147,16 @@ following availability:
 |Balances load between API master endpoints
 |===
 
+[NOTE]
+====
+The API Server load balancer HA must be ensured for production use. The unavailability of the 
```

Comment:
@acomabon, how does this sound? 

In production {product-title} clusters, you must maintain high unavailability
of the API Server load balancer. If the API Server load balancer is not
available, nodes cannot report their status, all their pods are marked dead,
and the pods' endpoints are removed from the service.

You must separately configure HA for the API Server load balancer. To configure
HA for it, you can run multiple HAProxy load balancers and use Keepalived to 
provide a constant virtual IP address for them.

Reply:
Thanks for your fast reply!
I'd say:

"In production {product-title} clusters, you must maintain high availability (not unavailability)
of the API Server load balancer...."

and in the last sentence:

"you can run multiple HAProxy load balancers and use Keepalived to
provide a floating (better word than constant) virtual IP address for them."

What do you think?

Reply:
I'm happy to help.
Both your suggestions sound good to me. :) Thank you for spotting my unfortunate typo!